build_test/lib/src/test_builder.dart

```diff
@@ -196,8 +200,16 @@ Future<TestBuilderResult> testBuilder(
 /// builder to run for all inputs. To use the default builder config instead,
 /// set [testingBuilderConfig] to `false`.
 ///
-/// Returns a [TestBuilderResult] with the [TestReaderWriter] used for
-/// the build, which can be used for further checks.
+/// Optionally pass [readerWriter] to pass in the filesystem that will be
```

Comment:
The two `pass`es makes it a little hard to read. Maybe, as above, something like `Optionally pass [ReaderWriter] to set the filesystem...`

Reply:
Done.